maven-plugin-plugin/src/it/mplugin-394_report-encoding/invoker.properties

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+invoker.goals = -Dfile.encoding=CP1252 plugin:report
```

Comment:
I know and am I bit late, but this is brittle and wrong. This system property shall only be set, at most, at JVM startup time. I highly doubt that Invoker does that.
Read JEP 400:

    Developers sometimes attempt to configure the default charset by setting the system property
    file.encoding on the   command line (i.e., java -Dfile.encoding=...), but this has never been supported.
    Furthermore, attempting to set the property programmatically (i.e., System.setProperty(...)) after the
    Java runtime has started does not work.

Reply:
Hallo @michael-o,

this was my reproduction of the problem (on a Windows system which has CP1252 as System-Encoding). My goal was to set the default encoding to CP1252, regardless of the system (Linux, Mac etc.). In hindsight you are right, that this test might not work as expected an non Windows machine (i.e. it never fails because the encoding is always UTF-8). Can you propose another way to safely accomplish a default/system/native encoding not equal to utf8 while running this integration test?

Reply:
honestly, we don't really write ITs for such encoding issues: there is no chance of regression at plugin level, only base encoding-aware XML reading class is well tested
so IMHO, don't worry too much, eventually remove the IT that was done if it's not useful: it was done based on good principles, but life is life on such feature

Reply:
@SebastianKuehn Please add a followup PR which documents the limitations of this PR